GitCommands/Git/GitItemStatusNameComparer.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace GitCommands.Git
+{
+    public class GitItemStatusNameComparer : IEqualityComparer<GitItemStatus>
+    {
+        public bool Equals(GitItemStatus x, GitItemStatus y)
+        {
+            return x?.Name == y?.Name;
+        }
+
+        public int GetHashCode(GitItemStatus obj)
+        {
+            return obj.Name?.GetHashCode() ?? 0;
```

Comment:
`obj` can too be `null`
```suggestion
            return obj?.Name?.GetHashCode() ?? 0;
```

Reply:
Not what I can see from the docs or in any example, but changing